src/cplus_plugin/gui/constant_raster_manager_dialog.py

```diff
@@ -258,15 +266,18 @@ def _register_sample_metadata(self):
         NCS pathways and activities.
         In production, metadata would be loaded from settings or defined elsewhere.
         """
-        from ..models.constant_raster import ConstantRasterMetadata, ConstantRasterCollection
+        from ..models.constant_raster import (
+            ConstantRasterMetadata,
+            ConstantRasterCollection,
+        )
 
         # Register for NCS Pathways
         if YEARS_EXPERIENCE_PATHWAY_ID not in self._raster_registry.metadata_ids():
             collection_pathway = ConstantRasterCollection(
                 filter_value=0.0,
                 total_value=1.0,  # Output range: 0-1 (normalized)
                 components=[],
-                skip_raster=False
+                skip_raster=False,
             )
 
             metadata_pathway = ConstantRasterMetadata(
```

Comment:
@osundwajeff `ConstantRasterMetadata` should have some default options when creating a new instance. Also, the `ConstantRasterWidgetInterface` should define a class method for creating a new instance of a `ConstantRasterMetadata` object. We can provide a default implementation but there is an option of overriding this default implementation.